assets/css/editor.scss

```diff
@@ -60,3 +60,26 @@
 	margin: 0;
 	padding: 0;
 }
+
+.theme-twentytwenty {
+	.wp-block {
+		h2.wc-block-grid__product-title {
+			@include font-size(large);
```

Comment:
[In `style.scss` font-size is set to `regular`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4545/files#diff-cd886530abfa495dc6782189ca779f1304bb73bc1758e5d3f0968448dfb67abeR221) but here it's `large`. We should try to unify those to use the same.

Reply:
I added this initially as the product titles in the editor are slightly smaller than the ones in the frontend when using the same font size. For consistency reasons, I'm using the same font size in the editor and frontend. Please let me know if the font size difference between the editor and the frontend is acceptable.